Python 3.13.0 (tags/v3.13.0:60403a5, Oct  7 2024, 09:38:07) [MSC v.1941 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите первую дробь (в формате a/b): 12
Введите вторую дробь (в формате a/b): 5
Сумма: 17
Разность: 7
Произведение: 60
Частное: 12/5

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите первую дробь (в формате a/b): 18
Введите вторую дробь (в формате a/b): 6
Сумма: 24
Разность: 12
Произведение: 108
Частное: 3

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите первую дробь (в формате a/b): 11
Введите вторую дробь (в формате a/b): 3
Сумма: 14
Разность: 8
Произведение: 33
Частное: 11/3

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите первую дробь (в формате a/b): 15
Введите вторую дробь (в формате a/b): 3
Сумма: 18
Разность: 12
Произведение: 45
Частное: 5

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Таймер запущен на 50 секунд...
Время вышло!

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Таймер запущен на 5 минута...
Время вышло!

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Сила тока: 0.7 А
Traceback (most recent call last):
  File "C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py", line 20, in <module>
    print(f"Напряжение: {calculated_voltage} В")
NameError: name 'calculated_voltage' is not defined. Did you mean: 'calculate_voltage'?

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Сила тока: 0.9 А
Traceback (most recent call last):
  File "C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py", line 20, in <module>
    print(f"Напряжение: {calculated_voltage} В")
NameError: name 'calculated_voltage' is not defined. Did you mean: 'calculate_voltage'?

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Сила тока: 0.5 А
Traceback (most recent call last):
  File "C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py", line 20, in <module>
    print(f"Напряжение: {calculated_voltage} В")
NameError: name 'calculated_voltage' is not defined. Did you mean: 'calculate_voltage'?

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Сила тока: 2.0833333333333335 А
Traceback (most recent call last):
  File "C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py", line 20, in <module>
    print(f"Напряжение: {calculated_voltage} В")
NameError: name 'calculated_voltage' is not defined. Did you mean: 'calculate_voltage'?

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Член 0: 1 * (3^5) * (4^0) = 0
Член 1: 5 * (3^4) * (4^1) = 240
Член 2: 10 * (3^3) * (4^2) = 720
Член 3: 10 * (3^2) * (4^3) = 720
Член 4: 5 * (3^1) * (4^4) = 240
Член 5: 1 * (3^0) * (4^5) = 0

Разложение (a + b)^n: None

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Член 0: 1 * (8^10) * (9^0) = 0
Член 1: 10 * (8^9) * (9^1) = 6480
Член 2: 45 * (8^8) * (9^2) = 51840
Член 3: 120 * (8^7) * (9^3) = 181440
Член 4: 210 * (8^6) * (9^4) = 362880
Член 5: 252 * (8^5) * (9^5) = 453600
Член 6: 210 * (8^4) * (9^6) = 362880
Член 7: 120 * (8^3) * (9^7) = 181440
Член 8: 45 * (8^2) * (9^8) = 51840
Член 9: 10 * (8^1) * (9^9) = 6480
Член 10: 1 * (8^0) * (9^10) = 0

Разложение (a + b)^n: None

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Член 0: 1 * (1^3) * (2^0) = 0
Член 1: 3 * (1^2) * (2^1) = 12
Член 2: 3 * (1^1) * (2^2) = 12
Член 3: 1 * (1^0) * (2^3) = 0

Разложение (a + b)^n: None

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
['а', 'б', 'в', 'г', 'д', 'е', 'ж', 'з', 'и', 'й', 'к', 'л', 'м', 'н', 'о', 'п', 'р', 'с', 'т', 'у', 'ф', 'х', 'ц', 'ч', 'ш', 'щ', 'ъ', 'ы', 'ь', 'э', 'ю', 'я']

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
list['abcdefghijklmnopqrstuvwxyz']

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Отсортированные данные (по возрасту): [{'name': 'Дарья', 'age': 18}, {'name': 'Виктория', 'age': 19}, {'name': 'Илья', 'age': 20}]

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Отсортированные данные (по возрасту): [{'name': 'Полина', 'age': 21}, {'name': 'Виктор', 'age': 25}, {'name': 'Михаил', 'age': 30}]

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Отсортированные данные (по возрасту): [{'name': 'Анастасия', 'age': 20}, {'name': 'Кирилл', 'age': 26}, {'name': 'Антон', 'age': 28}]

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Логарифм числа 33 по основанию 8: 1.6814647064528179

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Логарифм числа 12 по основанию 7: 1.276989408269624

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Логарифм числа 100 по основанию 2: 6.643856189774725

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Логарифм числа 50 по основанию 4: 2.821928094887362

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Enter a palindrome: ИИ
ИИ is palindrome!
Enter a palindrome: выход
not a palindrome
Enter a palindrome: казак
казак is palindrome!
Enter a palindrome: ага
ага is palindrome!
Enter a palindrome: аа
аа is palindrome!
Enter a palindrome: потоп
потоп is palindrome!
Enter a palindrome: ротор
ротор is palindrome!
Enter a palindrome: локатор
not a palindrome
Enter a palindrome: выход
not a palindrome
Enter a palindrome: 
= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите ваш возраст: 18
У вас есть разрешение? (да/нет): да
Доступ запрещен.

= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите ваш возраст: 16
У вас есть разрешение? (да/нет): нет
Доступ запрещен.
>>> 
= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите ваш возраст: 19
У вас есть разрешение? (да/нет): да
Доступ запрещен.
>>> 
= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите ваш возраст: 20
У вас есть разрешение? (да/нет): нет
Доступ запрещен.
>>> 
= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите ваш возраст: 13
У вас есть разрешение? (да/нет): нет
Доступ запрещен.
>>> 
= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Введите ваш возраст: 18
У вас есть разрешение? (да/нет): да
Вы можете войти.
>>> 
= RESTART: C:/Users/vlad_/AppData/Local/Programs/Python/Python313/My Project 5.py
Название: Гарри Поттер
Автор: Дж.К.Роулинг
Год издания: 1997
Содержание: Представляет собой хронику приключений юного волшебника Гарри Поттера, ......
Содержание книги обновлено.
Название: Гарри Поттер
Автор: Дж.К.Роулинг
Год издания: 1997
Содержание: посвящён противостоянию Гарри и тёмного волшебника по имени лорд Волан-де-Морт,...
